### ISSUE TITLE
bug: @ionic/angular-toolkit shows warnings in new project

### ISSUE THREAD

**Issue by josh-m-sharpe** [ionic-team/ionic#21652]
TLDR:
A freshly created ionic app using latest @ionic/cli results in warnings.   (Also happens on existing projects using these versions)

warning " > @ionic/angular-toolkit@2.2.0" has unmet peer dependency "@angular-devkit/architect@>=0.800.0".
(and a few others)

It seems that capacitor libs exhibit the same issue.

----------

ionic --version
6.10.1
yarn --version
1.22.4

ionic start testdeps tabs --capacitor --type=angular
cd testdeps
rm -rf node_modules package-lock.json
yarn install

yarn add @capacitor/ios @capacitor/android

yarn add v1.22.4
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
warning " > @ionic/angular-toolkit@2.2.0" has unmet peer dependency "@angular-devkit/architect@>=0.800.0".
warning " > @ionic/angular-toolkit@2.2.0" has unmet peer dependency "@angular-devkit/core@>=8.0.0".
warning " > @ionic/angular-toolkit@2.2.0" has unmet peer dependency "@angular-devkit/schematics@>=8.0.0".
warning "@ionic/angular-toolkit > copy-webpack-plugin@5.1.1" has unmet peer dependency "webpack@^4.0.0 || ^5.0.0".
warning " > @capacitor/ios@2.2.0" has incorrect peer dependency "@capacitor/core@~2.1.0".
warning " > @capacitor/android@2.2.0" has incorrect peer dependency "@capacitor/core@~2.1.0".
[4/4] 🔨  Building fresh packages...
success Saved lockfile.
success Saved 2 new dependencies.
info Direct dependencies
├─ @capacitor/android@2.2.0
└─ @capacitor/ios@2.2.0
info All dependencies
├─ @capacitor/android@2.2.0
└─ @capacitor/ios@2.2.0
✨  Done in 6.54s.
